crates/rolldown/tests/rolldown/warnings/invalid_option/invalid_output_dir_option/artifacts.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# warnings
+
+## INVALID_OPTION
+
+```text
+[INVALID_OPTION] Warning: Invalid value for option "output.dir" - you must set either "output.file" for a single-file build or "output.dir" when generating multiple chunks.
```

Comment:
If this gonna be a warning, I think we should add and underline the part "`output.dir` would be ignored."

Reply:
```suggestion
[INVALID_OPTION] Warning: Invalid value for option "output.dir" - "output.dir" is ignored. you must set either "output.file" for a single-file build or "output.dir" when generating multiple chunks.
```